tasks/install_shell.sh

```diff
@@ -389,6 +388,20 @@ install_file() {
         info "existing puppetlabs yum repo found, moving to old location"
         mv /etc/yum.repos.d/puppetlabs-pc1.repo /etc/yum.repos.d/puppetlabs-pc1.repo.old
       fi
+
+      if test "x$installed_version" != "xuninstalled"; then
```

Comment:
I guess having the `x` prepended is unnecessary. Will remove before merge (not going to force push right away so I don't change the ref in the bolt PR)